neo4j_utils/_query.py

```diff
@@ -29,4 +31,4 @@ class Query(
 
 
 Query.__new__.__defaults__ = (None,)
-Query.__annotations__ = {'query': str, 'args': dict | None}
+Query.__annotations__ = {'query': str, 'args': Optional[dict]}
```

Comment:
hm, unfortunately we have to support Py 3.8 for half more year, but this could be nicer with `typing.NamedTuple`, instead of `Optional` ([see example here](https://github.com/saezlab/pypath/blob/master/pypath/inputs/disgenet/_records.py))